Sources/Web3Auth/SessionManagement.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  File.swift
+//
+//
+//  Created by Dhruv Jaiswal on 18/07/22.
+//
+
+import Foundation
+import web3
+
+public class SessionManagement {
+    static let shared = SessionManagement()
+    private init() {}
+    private let storageServerUrl = "https://broadcast-server.tor.us"
+
+    func getActiveSession(sessionID: String) async throws -> Web3AuthState {
+        let publicKeyHex = SECP256K1.privateToPublic(privateKey: sessionID.web3.hexData!, compressed: false)!.web3.hexString.web3.noHexPrefix
```

Comment:
don't use ! anymore. The library is going to panic without appropriate errors. Some clients have complained about this in the past

Reply:
done